jsapp/xlform/src/view.acceptedFiles.coffee

```diff
@@ -0,0 +1,30 @@
+Backbone = require 'backbone'
+$baseView = require './view.pluggedIn.backboneView'
+$viewTemplates = require './view.templates'
+
+module.exports = do ->
+  class AcceptedFilesView extends $baseView
+    className: 'accepted-files card__settings__fields__field'
+    events: {
+      'input input': 'onChange'
+    }
+    placeholder: '.pdf,.doc,.odt,.docx'
```

Comment:
This is very elegant as a placeholder and an informative example, but after testing, this being the default just isn't so. https://github.com/XLSForm/pyxform/issues/194 specifies `application/*` as the default, so one of us misread something :)

Have any ideas for placeholder text that's better than `Enter a comma-separated list of extensions`?

Reply:
I tried finding the source of this default, but can't find it now 🤣 How about `e.g. ".pdf,.doc,.odt"` as an placeholder?